src/core/Akka.Persistence.TestKit.Xunit2/PersistenceTestKit.cs

```diff
@@ -0,0 +1,29 @@
+//-----------------------------------------------------------------------
+// <copyright file="PersistenceTestKit.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Akka.Persistence.TestKit
+{
+    using Akka.TestKit.Xunit2;
+
+    /// <summary>
+    /// This class represents an Akka.NET Persistence TestKit that uses <a href="https://xunit.github.io/">xUnit</a>
+    /// as its testing framework.
+    /// </summary>
+    public abstract class PersistenceTestKit : PersistenceTestKitBase
```

Comment:
Need to add a constructor overload here which takes an `ITestOutputHelper` for XUnit - that's how we're able to inject the XUnit output capture system into the Akka.NET logging infrastructure. Here's an example of a unit test where a user would do that:

https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka.Remote.Tests/RemotingSpec.cs#L31

Reply:
To correctly handle `ITestOutputHelper` in test kit I changed the base class of `PersistenceTestKit` to `TestKit`. This means that `PersistenceTestKitBase` no longer exists other test frameworks will not have Test Kit class with persistence testing API out of the box for now.